src/modules/claim/claim.controller.ts

```diff
@@ -121,10 +125,11 @@ export class ClaimController {
     );
 
     this.logger.debug(`credentials issued by ${did}`);
+    return result;
   }
 
   @Post('/request/:did?')
-  // @ApiExcludeEndpoint()
+  @ApiExcludeEndpoint()
```

Comment:
@Harasz Do you know why we had this commented before? It seems strange to me that it is excluded from the docs

Reply:
Yeah, last time I comment this line out and forgot to undo. So it's back to original state